model/task_lifecycle.go

```diff
@@ -1203,8 +1282,18 @@ func evalLinearStepback(ctx context.Context, t *task.Task, caller, status string
 
 // evalBisectStepback performs bisect stepback on the task.
 func evalBisectStepback(ctx context.Context, t *task.Task, caller, status string, deactivatePrevious bool) error {
-	// TODO: EVG-20788 implement stepback bisection.
-	return errors.Wrap(doBisectStepback(ctx, t), "performing bisect stepback")
+	// If the task is aborted or it isn't a mainline commit or existing stepback, no-op.
+	if t.Aborted || (t.ActivatedBy != evergreen.RepotrackerVersionRequester && t.ActivatedBy != evergreen.StepbackTaskActivator) {
+		return nil
+	}
+
+	newStepback := t.StepbackInfo == nil && evergreen.IsFailedTaskStatus(t.Status)
```

Comment:
passing in t.Status is fine, we should just remove the status param from this function now since it's unused. We still are gonna need caller and deactivatePrevious though so let's keep them for now (gonna create another ticket regarding this)